src/modules/0.0.22/aws_s3/entity/bucket_object.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryColumn,
+  PrimaryGeneratedColumn,
+  Unique,
+} from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+import { AwsRegions } from '../../aws_account/entity';
+import { Bucket } from './bucket';
+
+@Entity()
+@Unique('uq_bucketobject_id_region', ['id', 'region'])
+@Unique('uq_bucketobject_bucket_key_region', ['key', 'region'])
+export class BucketObject {
+  @PrimaryGeneratedColumn()
+  id?: number;
+
+  @PrimaryColumn({
+    nullable: false,
+    type: 'varchar',
+  })
+  @cloudId
+  key: string;
+
+  @cloudId
```

Comment:
not sure how this is behaving since it would be pointing 2 columns instead of one 🤔